bin/dupRadar.r

```diff
@@ -8,7 +8,14 @@ if (length(args) < 3) {
 input_bam <- args[1]
 annotation_gtf <- args[2]
 paired_end <- if(args[3]=='paired') TRUE else FALSE
-input_bam_basename <- strsplit(input_bam, "\\.")[[1]][1]
+
+bamRegex <- "(.*)\\.bam"
```

Comment:
Maybe `(.+)\\.bam$` to have at least one character before the first `.` and that the filename ends with `.bam`?

Reply:
Good point !